README.rst

```diff
@@ -40,6 +40,35 @@ The AMD Inference Server is integrated with the following libraries out of the g
 * XModel models with `Vitis AI <https://www.xilinx.com/products/design-tools/vitis/vitis-ai.html>`__ on AMD FPGAs
 * A graph of computation including as pre- and post-processing can be written using `AKS <https://github.com/Xilinx/Vitis-AI/tree/v2.5/src/AKS>`__ on AMD FPGAs for end-to-end inference
 
+Quick Start Inference
+----------------------------------------
+
+Quick start requires AMD EPYC CPU::
+
+  # Step 1: Create the example model repository(requires git-lfs)
```

Comment:
This code sample should be under a `.. code-block:: bash` directive. Look at [other docs with code snippets](https://github.com/rradjabi/inference-server/blob/quickstart/docs/zendnn.rst#build-the-image) for reference